packages/react/src/faChar.tsx

```diff
@@ -0,0 +1,20 @@
+import { Utility } from "@hpcc-js/common";
+import * as React from "@hpcc-js/preact-shim";
+
+interface FAChar {
+    x?: number;
+    y?: number;
+    height?: number;
+    fill?: string;
+    stroke?: string;
+    faChar?: string;
+}
+
+export const FAChar: React.FunctionComponent<FAChar> = ({
+    x,
+    y = 0,
+    height = 12,
+    fill,
+    stroke,
+    faChar = ""
+}) => <text x={x} y={y - height * 3.5 / 12} fill={fill} stroke={stroke} font-family="FontAwesome" font-size={`${height}px`} style="text-anchor: middle;" >{Utility.faChar(faChar)}</text>;
```

Comment:
This `y` might be a bit off for some of the fa icons. Maybe we should add another utility method that precisely measures the pixel height and offset of a characters?

Reply:
@jbrundage textSize should probably work and we could pre-calc all their actual sizes @12px and store with the name/char map we already have?